PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/serverexplore/ui/SparkServerlessProvisionDialog.java

```diff
@@ -76,13 +76,13 @@
     protected JLabel workerCoresLabel;
     protected JLabel workerMemoryLabel;
     protected JLabel workerNumberOfContainersLabel;
-    protected JTextField errorMessageField;
     protected JPanel provisionDialogPanel;
     protected JButton refreshButton;
     protected JLabel storageRootPathLabel;
     protected JComboBox sparkVersionComboBox;
     protected JLabel sparkVersionLabel;
     private JXHyperLinkWithUri jobQueueHyperLink;
+    protected JTextArea errorMessageTextArea;
```

Comment:
Is it scrollable?

Reply:
Made it scrollable in the new commit.

---
In reply to: [210529202](https://github.com/Microsoft/azure-tools-for-java/pull/1879#discussion_r210529202) [](ancestors = 210529202)